production/kubernetes/agent-tempo.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: grafana-agent-traces
+---
+apiVersion: v1
+data:
+  agent.yaml: |
+    server:
+        http_listen_port: 8080
+        log_level: info
+    tempo:
+        push_config:
+            basic_auth:
+                password: ${TEMPO_PASSWORD}
+                username: ${TEMPO_USERNAME}
+            batch:
+                send_batch_size: 1000
+                timeout: 5s
+            endpoint: ${TEMPO_ENDPOINT}
+            queue:
+                retry_on_failure: false
+        receivers:
+            jaeger:
+                protocols:
+                    grpc: null
+                    thrift_compact: null
+                remote_sampling:
+                    insecure: true
+                    strategy_file: /etc/agent/strategies.json
+        scrape_configs:
+          - bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+            job_name: kubernetes-pods
+            kubernetes_sd_configs:
+              - role: pod
+            relabel_configs:
+              - action: replace
+                source_labels:
+                  - __meta_kubernetes_namespace
+                target_label: namespace
+              - action: replace
+                source_labels:
+                  - __meta_kubernetes_pod_name
+                target_label: pod
+              - action: replace
+                source_labels:
+                  - __meta_kubernetes_pod_container_name
+                target_label: container
+            tls_config:
+                ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+                insecure_skip_verify: false
+  strategies.json: '{"default_strategy": {"param": 0.001, "type": "probabilistic"}}'
```

Comment:
i guess this is a good idea?  what is the mechanism for overriding this?  

if the user can't customize this it may make more sense to just leave it out.

Reply:
If they're running with the install script the only real option is manually editing the config map, but if they're using the Tanka configs they'd override it [here](https://github.com/rfratto/agent/blob/f484ff5190f8be994c928646447e7c52813dc3ab/production/kubernetes/build/templates/tempo/main.jsonnet#L31-L36).